js/page-specific/phone-directory/phone-directory.js

```diff
@@ -2,107 +2,102 @@ var jQuery = $.noConflict(true);
+  }
+  var newPageNum = currentPageNum + offset;
+  document.getElementById("Page" + currentPageNum).style.display = "none";
+  document.getElementById("Page" + newPageNum).style.display = "block";
 
-    jQuery('#Next').toggle(newPageNum < pageCount);
-    jQuery('#Previous').toggle(newPageNum > 1);
+  jQuery("#Next").toggle(newPageNum < pageCount);
+  jQuery("#Previous").toggle(newPageNum > 1);
 }
 
 function validateForm() {
-    var firstName = jQuery('#txtFirst').val().trim();
-    var lastName = jQuery('#txtLast').val().trim();
-    if (firstName == "" && lastName == "") {
-        alert(
-            "If Searching all Agencies, You must enter at least one letter of the last name or one letter of the first name"
-        );
-        return false;
-    }
-    return true;
+  var firstName = jQuery("#txtFirst").val().trim();
+  var lastName = jQuery("#txtLast").val().trim();
+  if (firstName == "" && lastName == "") {
+    alert(
+      "If Searching all Agencies, You must enter at least one letter of the last name or one letter of the first name"
+    );
+    return false;
+  }
+  return true;
 }
 
 jQuery(document).ready(function () {
-    jQuery.support.cors = true; // force cross-site scripting (as of jQuery 1.5)
-
-    jQuery("#btnSubmit").click(function (submitEvent) {
-        submitEvent.preventDefault();
-
-        //Hide next previous button incase they were visible from a previous search
-        jQuery('#Next,#Previous').hide();
-        if (validateForm()) {
-            jQuery("#output").text("");
-            dataString =
-                jQuery(this).closest("form").serialize() +
-                "&formId=" +
-                jQuery(this).closest("form").attr("id");
-
-            jQuery.ajax({
-                url: "https://phonedirectorydev.baltimorecountymd.gov/PhoneDir/ProcessPhoneDirSearchForm",
-                data: dataString,
-                type: "GET",
-                dataType: "jsonp",
-                jsonpCallback: "formatJsonpResult",
-            });
-        } //end of if ValidForm
-    });
-
-    jQuery("#btnClear").click(function (clearEvent) {
-        clearEvent.preventDefault();
-        jQuery("#BACO_table,#prevNext,#output center").hide();
-        jQuery("#txtLast,#txtFirst").val("");
-        jQuery("#ddlAgency").val("0");
-    });
+  jQuery.support.cors = true; // force cross-site scripting (as of jQuery 1.5)
+
+  jQuery("#btnSubmit").click(function (submitEvent) {
+    submitEvent.preventDefault();
+
+    //Hide next previous button incase they were visible from a previous search
+    jQuery("#Next,#Previous").hide();
+    if (validateForm()) {
+      jQuery("#output").text("");
+      dataString =
+        jQuery(this).closest("form").serialize() +
+        "&formId=" +
+        jQuery(this).closest("form").attr("id");
+
+      jQuery.ajax({
+        url:
+          "https://testservices.baltimorecountymd.gov/api/hub/phoneDirectory/ProcessPhoneDirSearchForm",
```

Comment:
@srutherfordbcg this is the important change here, the rest is just a reformat.

Reply:
Looks pretty straight forward to me.